src/app/app-routing.module.ts

```diff
@@ -13,6 +13,13 @@ const routes: Routes = [
           import('./pages/home/home.module').then((m) => m.HomeModule),
       },
       { path: '', redirectTo: 'home', pathMatch: 'full' },
+      {
+        path: 'listings',
+        loadChildren: async () =>
+          await (
+            await import('./pages/listings/listings.module')
+          ).ListingsModule,
+      },
```

Comment:
Can we do it using .then for consistency

Reply:
Yes Done.